RoadMap

```diff
@@ -0,0 +1,25 @@
+StartDate - 15th Aug 2019
+EndDate - By End of 2019
+
+List of Data Structures & Algorithms : 4 months ( 18 weeks) : 
+
+Array (2 weeks)
+Linked List ( 1 week)
+Stacks and Queues : 2 weeks
+Trees : 3 weeks
+Heaps : 1 week
+Graphs : 1 week
+HashMap : 1 week
+Divide and conquer : 1 week
+String algorithms : 1 week
+Sorting & Searching : 1 week
+Backtracking & Recursion : 2 weeks
+Dynamic programming : 2 weeks
+
+This includes regular practice in hackerrank
+Source : karumanchi book and geeksforgeeks
+
+Design :
+Design Patterns - Head First Design patterns book (45 days tentative)
```

Comment:
Will the Design pattern go separately after DS. Or it will progress in parallel. Can we draw a roadmap for Design as well?

Reply:
for 45 days we will follow the head first book and complete it.
I will put the roadmap for system design after gathering some info

Reply:
Ok